x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/execution_log_table/execution_log_columns.tsx

```diff
@@ -139,6 +156,34 @@ export const getMessageColumn = (width: string) => ({
   width,
 });
 
+export const getSourceEventTimeRangeColumns = () => [
+  {
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_SOURCE_EVENT_TIME_RANGE}
+        tooltipContent={i18n.COLUMN_SOURCE_EVENT_TIME_RANGE_TOOLTIP}
+      />
+    ),
+    field: 'backfill',
+    render: (backfill: { to: string; from: string }) => {
+      return backfill ? (
+        <div>
+          <div>
+            <FormattedDate value={backfill.to} fieldName="backfill.to" />
+          </div>
+          <EuiText textAlign="center">{'-'}</EuiText>
+          <div>
+            <FormattedDate value={backfill.from} fieldName="backfill.from" />
```

Comment:
Hm, I think it's what usually expected as we use this component to show dates across all security solution.

Also if somebody else make manual rule run and I live in different time zone, I probably want to have this range shown in mine, as it easy for me understand when it happen, but also I can use this range in different places in kibana

Reply:
Yeah, it probably felt weird because I used the API directly and saw the different values in the table. But since our interface always displays time in the user's time zone, that should be fine 👍